packages/ts/react-crud/src/autogrid.tsx

```diff
@@ -88,6 +97,20 @@ interface AutoGridOwnProps<TItem> {
    * grid.
    */
   rowNumbers?: boolean;
+  /**
+   * When enabled, shows the total count of records in the grid footer.
+   */
+  totalCount?: boolean;
+  /**
+   * When enabled, shows the visible record count in the grid footer.
```

Comment:
I suggest we rename the property to `filteredCount` and only show it when there is a filter applied to the grid. To be clear, as the number of filtered records may exceed the visible viewport of the grid, IMO the end user wants to see how many records (out of total) are affected by the applied filters, no matter how the UI is handling the pagination (infinite scrolling, or paging navigation buttons or etc.)
 
I'm not sure if it is doing the same thing now, as I couldn't find out whether infinite scrolling and pageSize are addressing something related to filtered count, or that part of the code is addressing something else. 
```suggestion
   * When enabled, shows the filtered record count in the grid footer.
```